lib/lld-wrapper/src/link.cc

```diff
@@ -0,0 +1,17 @@
+#include <cstdint>
+
+#include <llvm/Support/raw_ostream.h>
+
+#include <lld/Common/Driver.h>
+#include <lld/Common/ErrorHandler.h>
+
+extern "C" {
+
+void wasmer_lld_wrapper_link(const char **filenames, uint32_t count) {
+  llvm::ArrayRef<const char *> files(filenames, count);
+
+  if (!lld::elf::link(files, false, llvm::outs(), llvm::errs()))
+    lld::exitLld(0);
```

Comment:
What this command will do? (exit the process?)

Reply:
Yep. To quote the docs:
```
// Generic entry point when using LLD as a library, safe for re-entry, supports
// crash recovery. Returns a general completion code and a boolean telling
// whether it can be called again. In some cases, a crash could corrupt memory
// and re-entry would not be possible anymore. Use exitLld() in that case to
// properly exit your application and avoid intermittent crashes on exit caused
// by cleanup.
```

We probably just want to not use this API at all, but for now this is how it's going to be.